### DISCUSSION TITLE
Build kens in Ubuntu operated in aarch64 architecture

### DISCUSSION THREAD

**Discussion by JYBESiU** [ANLAB-KAIST/KENSv3#231] [Project Q&A 4. Etc] [answered]
I want to build my kens project in Linux environment, so I try to use UTM to operate Ubuntu 20.04 version in my ARM64 CPU(M2) Macbook.

However, when I try to build with `cmake` command, I found there is no kens-solution file.
![스크린샷 2024-05-12 오후 2 44 21](https://github.com/ANLAB-KAIST/KENSv3/assets/39625161/1e113046-8516-46d5-9ad4-c9f72ff1684c)

Is it impossible to build kens in my situation?

**Comment by Pusnow** [comment, marked as answer]
Currently KENS does not support Linux/ARM environments. Please use the other environments.
